engine/servermaster/openapi.go

```diff
@@ -76,10 +83,15 @@ func NewOpenAPI(infoProvider ServerInfoProvider) *OpenAPI {
 // RegisterOpenAPIRoutes registers routes for OpenAPI.
 func RegisterOpenAPIRoutes(router *gin.Engine, openapi *OpenAPI) {
 	v1 := router.Group("/api/v1")
-	v1.Use(openapi.ForwardToLeader)
 	v1.Use(openapi.httpErrorHandler)
 
+	v1.GET("/status", openapi.ServerStatus)
+
+	leaderGroup := v1.Group("/leader")
+	leaderGroup.POST("/resign", openapi.ResignLeader)
```

Comment:
What happens when calling `/leader/resign` on a non-leader server master.

Reply:
One more question. `resign` won't take effect any more after a restart, right?

Reply:
`resign` only takes effect when the sever is a leader.